tokenomics.md

```diff
@@ -27,23 +27,21 @@ Token allocation is benchmarked to similar successful projects. Typically, such
+One point to note on the listing above is the allocation to initial funders. They include the initial developer of Obscuro.  They receive an allocation of the Obscuro token as payment in kind for the work done to date on the project. This is a one-time only allocation, after which funders will not be involved in either development or day to day operations of the Obscuro Foundation. The number of tokens allocated to funders is to be agreed upon by the Obscuro Foundation.
 
 ## Relationship to ETH
-As each Obscuro block is published to the Ethereum Mainnet, Obscuro aggregators must hold a balance of ETH to be able to do this. Aggregators get paid in OBX (via block reward and transaction fees) and have outgoings in ETH, therefore there exists currency exposure between OBX and ETH, which, in theory, can be hedged with futures or options if such a market existed. The combination of transaction fees and block reward in terms of ETH must be great enough to:
+As each Obscuro block is published to the Ethereum Mainnet, Obscuro aggregators must hold a balance of ETH to publish. Aggregators get paid in OBX (via block reward and transaction fees) and have outgoings in ETH; therefore, there exists currency exposure between OBX and ETH, which, in theory, can be hedged with futures or options if such a market existed. The combination of transaction fees and block reward in terms of ETH must be large enough to:
 * Cover the ETH required to publish the latest Obscuro block to Ethereum mainnet.
 * Cover the Obscuro computation required to validate and publish the block.
 * Cover a profit margin for the Obscuro aggregators.
 
-It is thus important for the OBX/ETH price to remain as stable as possible. The incentives of the various groups involved in the network should be able to facilitate this. To participate in the network or become an aggregator, users must buy OBX tokens. On the flip side, aggregators will likely sell OBX to cover their ETH expenses. These flows coupled with large amounts of OBX locked up in staking arrangements should enable OBX to retain enough value versus ETH to ensure being an aggregator is profitable.
+It is thus essential for the OBX/ETH price to remain as stable as possible. The incentives of the various groups involved in the network should be able to facilitate this. To participate in the network or become an aggregator, users must buy OBX tokens. On the flip side, aggregators are likely to sell OBX to cover their ETH expenses. These flows coupled with large amounts of OBX locked up in staking arrangements should enable OBX to retain enough value versus ETH to ensure being an aggregator is profitable.
```

Comment:
"It is thus essential for the OBX/ETH relationship to remain as stable as possible."
"To participate in the network or become an Aggregator, users must have OBX tokens to execute transactions"

Remove the "On the flip side... to profitable" piece. Rather than explaining how aggregators sell OBX can we instead give a one liner to say "Aggregator behaviour is incentivised by a fees and rewards structure". Then link the Fees and Rewards section.

Reply:
not just execute transaction but stake too (for Aggregators)